pages/examples/checklist-plots.py

```diff
@@ -0,0 +1,58 @@
+import plotly.express as px
+from dash import Dash, dcc, html
+from dash import Output, Input
+from plotly import data
+
+# Data
+df = data.iris()
+
+species = df["species"].unique().tolist()
+
+options = [
+    {'label': specie.capitalize(),
+     'value': specie} for specie in species]
+
+fig = px.scatter(
```

Comment:
can you tell me more please why you chose to build the fig here as well as in the callback